llama_index/__init__.py

```diff
@@ -138,6 +141,7 @@
 
 
 __all__ = [
+    "OpenAI",
```

Comment:
i'd prefer to not add any more root imports, importing base package is already really slow